src/main/java/gregtech/common/metatileentities/multi/multiblockpart/appeng/MetaTileEntityMEStockingBus.java

```diff
@@ -64,9 +64,22 @@ protected ExportOnlyAEStockingItemList getAEItemHandler() {
     @Override
     public void update() {
         super.update();
-        if (!getWorld().isRemote && isWorkingEnabled() && autoPull && getOffsetTimer() % 100 == 0) {
-            refreshList();
-            syncME();
+        if (!getWorld().isRemote) {
+            if (isWorkingEnabled() && autoPull && getOffsetTimer() % 100 == 0) {
+                refreshList();
+                syncME();
+            }
+
+            // Immediately clear cached items if the status changed, to prevent running recipes while offline
+            if (this.meStatusChanged && !this.isOnline) {
+                if (autoPull) {
+                    clearInventory(0);
```

Comment:
Can this use `getAEItemHandler().clearConfig()`? If so maybe this and other occurrences of `clearInventory(0)` should move to it.

Reply:
They do basically the same thing, so I don't see a particular reason to switch.